### ISSUE TITLE
"Back to channel" button open.... avatar :)

### ISSUE THREAD

**Issue by TurboPlanner** [UnigramDev/Unigram#2718]
**Describe the bug**
"Back to channel" button is not working. It opens avatar

**To Reproduce**
1. Open comments to channel message
2. Try to press "back to channel" button in upper left corner
3. Unigram open channel avatar

**Expected behavior**
Back to channel from comments instead of open a avatar

**Screenshots**
![avant](https://user-images.githubusercontent.com/92896692/165084021-925f790b-3fa4-4d77-b7a9-ca2af62e7650.jpg)

**Version Info**
Version of Unigram - 8.7 (7352) I have auto update to new version

OS
Edition Windows 10 Enterprise
Version 21H1
OS build 19043.1586
Experience Windows Feature Experience Pack 120.2212.4170.0
Updated to last cumulative fixes

**Comment by makisukurisu** [comment]
It's not a back to channel button, it's a "replies" or "comments" icon.

Back button is located at the top left corner of the app from now on.
Opening a profile picture is still a bug, but AFAIK it's known by developers.

Thanks for your report!
Feel free to comment, and to reopen the issue anytime.

**Comment by MarcAnt01** [comment]
This is true, but I still think the ui should be changed, since it literally looks like a back button
![image](https://user-images.githubusercontent.com/43090263/165085644-ba840896-3aa9-45d5-92b0-f458f2f5d0a9.png)

**Comment by TurboPlanner** [comment]
> This is true, but I still think the ui should be changed, since it literally looks like a back button ![image](https://user-images.githubusercontent.com/43090263/165085644-ba840896-3aa9-45d5-92b0-f458f2f5d0a9.png)

No, it's not UI bug. It's functional bug and blocking issue if you are not subscriber of the channel.

For example, some channel or user forward message from another channel2. You open original message in the channel2 for reading comments. However you have not any way to return to original message if you not subscribe to channel2. In most cases users don't subscribe to channels in forwarded messages. Therefore Unigram has blocking bug for browsing non-subscribed channels. It's really critical because it is main use case with channels.

**Comment by FrayxRulez** [comment]
Did you try to click here?
![image](https://user-images.githubusercontent.com/8152849/165088217-fe264bef-6f74-4251-aacd-716bbf9841a1.png)

Just wondering...

**Comment by TurboPlanner** [comment]
> Did you try to click here? ![image](https://user-images.githubusercontent.com/8152849/165088217-fe264bef-6f74-4251-aacd-716bbf9841a1.png)
> 
> Just wondering...

Uppss.... Haha. Sorry :)

However I think that "big back button" should work as back button. Yes, it's UI bug

**Comment by MarcAnt01** [comment]
If not, the ux should be made more straighforward to make it clear that clicking there leads to the channel pic

**Comment by makisukurisu** [comment]
Perhaps it should be an "enhancement" then, and maybe update of the title/reopening a new issue would be good too?

**Comment by FrayxRulez** [comment]
Done
